prometheus-app.opam

```diff
@@ -32,10 +32,12 @@ depends: [
   "re"
   "cohttp-lwt" {>= "4.0.0"}
   "cohttp-lwt-unix" {>= "4.0.0"}
-  "cohttp-async" {>= "4.0.0"}
+  # We need the change of cohttp-async here:
+  # https://github.com/mirage/ocaml-cohttp/blob/e5a66f1c1e7c2e5051723e09260222994dff40cf/CHANGES.md?plain=1#L73
+  "cohttp-async" {= "6.0.0~alpha2"}
```

Comment:
Wouldn't that change be available in v6.0.0~beta1 as well?

Reply:
Ah. They made a beta release after i tested it on `alpha2`. i'm not sure if `>=` works with non-semantic versioning, but i can make the change.

Reply:
I looked up [how versioning works for opam](https://opam.ocaml.org/doc/Manual.html#version-ordering) and it turns out the ordering would be `6.0.0~alpha2` < `6.0.0~beta1` < `6.0.0`. So i have relaxed the version of `cohttp-async` to be lower bound to `6.0.0~alpha2` to be most accommodating as a library.